src/System.Text.Http.Parser/HttpParser.cs

```diff
@@ -323,7 +323,7 @@ public HttpParser(bool showErrorDetails)
                                 var current = reader.Position;
                                 var subBuffer = buffer.Slice(reader.Position);
                                 // Split buffers
-                                var lineEnd = subBuffer.PositionOf(ByteLF);
+                                var lineEnd = ReadOnlyBufferExtensions.PositionOf(subBuffer, ByteLF);
```

Comment:
?

Reply:
It compares two implementations of `PositionOf` on ISequence and ROB, now when both of them are extension methods invocation is ambiguous

Reply:
Isn't this a problem?

Reply:
No, the other extension method doesn't ship it's in buffers.Experimental